plugins/azure/azure.go

```diff
@@ -74,6 +74,10 @@ func Register(clusters *clusters.Clusters, plugins *plugin.Plugins, config Confi
 			containerInstancesRouter.Put("/containergroup/restart", router.restartContainerGroup)
 		})
 
+		r.Route("/costmanagement", func(costManagementRouter chi.Router) {
+			costManagementRouter.Get("/actualCost", router.getActualCost)
```

Comment:
Can you change this to lowercase please:

```suggestion
			costManagementRouter.Get("/actualcost", router.getActualCost)
```